### DISCUSSION TITLE
Buttons not available

### DISCUSSION THREAD

**Discussion by palengh** [astrandb/miele#120] [General]
Hi, there, and a big `thank you` to Åke (@astrandb) for this integration!

All sensors seems to work, but I cannot get the two buttons to become available (listed _unavailable_). 
Does the machine need to be in a special state for it to be available in HA?

Both `button.washing_machine_start` and `button.washing_machine_stop` _unavailable_

**Comment by mundschenk-at** [comment]
It's a Miele API problem. The buttons used to work fine with my washer-dryer, but not anymore. Either Miele has changed their API or something broke again. Unfortunately, they document neither.
